networking/multiple_networks/secondary_networks/configuring-multi-network-policy.adoc

```diff
@@ -6,8 +6,12 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-Multi-network policy is used by administrators to manage secondary networks created using the network attachment definition (NAD) resource. Administrators use the `MultiNetworkPolicy` API to manage traffic for pods that are attached to multiple networks. For example, you can separate the control plane and data plane traffic.
+Administrators can use the `MultiNetworkPolicy` API to create multiple network policies that manage traffic for pods attached to secondary networks. For example, you can manage traffic between pods attached to the same or different secondary networks by creating policies that allow or deny traffic based on specific ports or labels.
```

Comment:
no this is not accurate :( @dougbtv please help here as well.
@stevsmit this is not about same or different secondary networks... checkout the MNP API to see what that API does? its about ensuring users can create policies on secondary networks..
@maiqueb didn't we do some feature docs for MNP ? PTAL

Reply:
I'm just going to revert to the original and keep the note [here](https://github.com/openshift/openshift-docs/pull/84239#discussion_r1888522450). We can fix or add stuff later.

Reply:
no I was the stupid one here: https://github.com/k8snetworkplumbingwg/multi-networkpolicy?tab=readme-ov-file#policy-for-annotation we can have an array of networks the MNP can apply to.
@stevsmit you are right. I am the one wrong here.

Reply:
yes, but, still, I'd say the sentence reads weird / is misleading. 

Like the following part:
> For example, you can manage traffic between pods attached to the same or different secondary networks by creating policies that allow or deny traffic based on specific ports or labels

If these networks are disconnected, no amount of policy will make them "connected". And I kind of understand that from the text ?... 

I suggest we "write less". The following sentence is clearly correct, and crystal clear (IMHO...):
> Administrators can use the `MultiNetworkPolicy` API to create multiple network policies that manage traffic for pods attached to secondary networks.

And I'd say the next one (slight changes from the original proposal) is less misleading .
> For example, you can manage traffic between pods over their secondary network attachments by creating policies that allow or deny traffic based on specific ports, IPs or pod/namespace labels.